docs/synapse/userguides/ug013_storm_ref_lift.rst

```diff
@@ -191,12 +197,18 @@ Lifts a set of nodes based on one or more tags.
   **#** *<tag>* ...
   
 **Examples:**
+
 *Lifts all nodes that have the tag foo.bar or the tag baz.faz.*
 ::
   alltag( foo.bar , baz.faz )
   
   #foo.bar #baz.faz
 
+*Lifts up to three nodes that have the tag foo.bar*
+::
+  alltag( foo.bar , limit=3)
+
+
 **Usage Notes:**
 
 * ``alltag()`` retrieves all nodes that have **any** of the specified tags.
```

Comment:
Consider adding a second note (bullet) to specifically call out that limit (as a parameter) is not supported in macro syntax, e.g.:

* The macro syntax for alltags() does not support the use of a limit parameter with the operator itself. The limit() operator [link] can be used with the alltags() macro syntax to limit the number of nodes returned: #foo.bar limit(10).

Reply:
If we add this note, should we also note the subtle difference that the second query will lift all the tags then filter down, vs just lifting that number of nodes?

Reply:
fixed

Reply:
deleting